opus/import/obs_bundle_cassini_uvis_solarocc_beckerjarmak2023.py

```diff
@@ -0,0 +1,246 @@
+################################################################################
+# obs_bundle_cassini_uvis_solarocc_beckerjarmak2023.py
+#
+# Defines the ObsBundleCassiniUvisSolarOccBeckerJarmak class, which encapsulates
+# fields in the common, common occultation, obs_mission_cassini, and obs_instrument_couvis
+# tables for COUVIS solar occultations. This class supports derived data from UVIS EUV
+# archived in cassini_uvis_solarocc_beckerjarmak2023.
+################################################################################
+
+from datetime import datetime, date
+from obs_bundle_occ_common import ObsBundleOccCommon
+from obs_cassini_common_pds4 import ObsCassiniCommonPDS4
+
+class ObsBundleCassiniUvisSolarOccBeckerJarmak(ObsBundleOccCommon, ObsCassiniCommonPDS4):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    def _inst_name(self):
```

Comment:
`_inst_name` is something that was unique to the Uranus bundle because it has different instruments. It's not needed for this bundle and is not referenced in this file. It should be deleted.